src/components/WishlistPage/WishlistItem/WishlistItem.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import propTypes from 'prop-types';
+import cx from 'classnames';
+
+import './WishlistItem.scss';
+
+const CN = 'wishlist-item';
+
+class WishlistItem extends Component {
+  handleRemoveFromWishlist() {
+    const { action, item } = this.props;
+    action(item);
+  };
+
+  render() {
+    const { item } = this.props;
+    const { photo, title, collection, category } = item;
```

Comment:
You can write here 
```suggestion
    const { photo, title, collection, category } = this.props.item;
```
no need to create redundant constant